db/migrate/20201029155930_add_timezone_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimezoneToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :time_zone, :string, null: false, default: 'Eastern Time (US & Canada)'
```

Comment:
Don't set the default in the schema. Instead, set a callback in the user model, and make the default based on `config.time_zone`.